src/structures/applicationCommand.ts

```diff
@@ -113,16 +113,17 @@ export class ApplicationCommandInteraction extends Interaction {
   option<T>(name: string): T {
     const options = this.options
     const op = options.find((e) => e.name === name)
-    if (op === undefined || op.value === undefined) return undefined as any
+    if (op === undefined || op.value === undefined)
+      return undefined as unknown as T
```

Comment:
maybe set the return type as `T | undefined`

Reply:
as discussed on discord, i guess this is resolved?

Reply:
yeah i guess